src/os/im/action/importactionmanager.js

```diff
@@ -351,24 +358,72 @@ os.im.action.ImportActionManager.prototype.getEntryItems = function(type) {
 
 /**
  * Executes enabled import action entries of a type against a set of items.
- *
  * @param {string} entryType The entry type.
- * @param {Array<T>=} opt_items The items to process.
+ * @param {Array<T>} items The items to process.
  * @param {boolean=} opt_unprocess Reset existing items
+ * @param {boolean=} opt_unprocessOnly Do not process enabled entries
+ * @private
  */
-os.im.action.ImportActionManager.prototype.processItems = function(entryType, opt_items, opt_unprocess) {
-  var items = opt_items || this.getEntryItems(entryType);
+os.im.action.ImportActionManager.prototype.processItemsInternal_ = function(
+    entryType,
+    items,
+    opt_unprocess,
+    opt_unprocessOnly) {
   if (items && items.length > 0) {
+    var configs = [];
     var entries = this.actionEntries[entryType];
     if (entries && entries.length > 0) {
       for (var i = 0; i < entries.length; i++) {
-        if (entries[i].isEnabled()) {
-          entries[i].processItems(items);
-        } else if (opt_unprocess) {
-          entries[i].unprocessItems(items);
+        var cfgs = null;
+        if (!opt_unprocessOnly && entries[i].isEnabled()) {
+          cfgs = entries[i].processItems(items);
+        } else if (opt_unprocess || opt_unprocessOnly) {
+          cfgs = entries[i].unprocessItems(items);
+        }
+        if (cfgs) {
+          cfgs.forEach((cfg) => {
+            configs.push(cfg);
+          });
         }
       }
     }
+
+    if (configs.length > 0) {
```

Comment:
The following seems to be geared around items being `ol.Feature` instances. Should this be in `plugin.im.action.feature.Manager`?